src/test/resources/testng_suites/web-saucedemo.xml

```diff
@@ -1,24 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
-<suite name="saucedemo test">
-    <test name="test login">
+<suite verbose="1" name="Saucedemo test" parallel="methods">
+    <test name="Login Test">
         <classes>
-            <class name="com.solvd.meta.carina.LoginTest"/>
+            <class name="com.solvd.meta.carina.LoginTest">
+                <methods>
+                    <include name="loginTest"></include>
```

Comment:
You can make it shorter:
`<include name="loginTest"/> `instead of `<include name="loginTest"></include>`